src/components/content-tab-snapshots.tsx

```diff
@@ -273,7 +273,7 @@ function EmptyGeneric( {
 		<div className="p-8 flex justify-between max-w-3xl gap-4">
 			<div className="flex flex-col">
 				<div className="a8c-subtitle mb-1">{ __( 'Share a demo site' ) }</div>
-				<div className="w-[40ch] text-a8c-gray-70 a8c-body pr-2">
```

Comment:
This did not seem to do anything for `ltr` or `rtl` languages so I just removed it